nutils/sample.py

```diff
@@ -245,13 +245,13 @@ def __call__(self, __func: function.IntoArray) -> function.Array:
         indices = evaluable.loop_concatenate(evaluable._flat(self.get_evaluable_indices(ielem)), ielem)
         return _ReorderPoints(_ConcatenatePoints(func, self), indices) * funcscale
 
-    def basis(self) -> function.Array:
+    def basis(self, interpolation: str = 'none') -> function.Array:
```

Comment:
Can you document the `interpolation` argument, e.g. by referring to `asfunction`?